### ISSUE TITLE
Make tests excluded or forced

### ISSUE THREAD

**Issue by basharov** [franela/goblin#56]
It would be great to have excluding of tests, for example, by renaming `It` to `Xit`:

```
    // This test will be excluded and not run
     g.Xit("Should add two numbers ", func() {
            g.Assert(1+1).Equal(2)
        })
```

And vice versa, force a single test or test suite to run by renaming `It` to `Fit`:

```
    // Only this tests will run 
     g.Fit("Should add two numbers ", func() {
            g.Assert(1+1).Equal(2)
        })
```

Thanks!

**Comment by marcosnils** [comment]
@basharov I like the Xit feature, in fact we though about implementing it but we didn't have time. 

Regarding the Fit thing, wouldn't `goblin.run` flag work?

**Comment by marcosnils** [comment]
Ping

**Comment by marcosnils** [comment]
Fixed by #62
